create_test_cluster.py

```diff
@@ -92,7 +110,8 @@ def check_args() -> Namespace:
     description="This tool can be used to install the Stackable Kubernetes Operators into a Kubernetes cluster using Helm. "
                 "It can optionally also create a kind cluster."
   )
-  parser.add_argument('--operator', '-o', help='A list of Stackable operators to install. Operators can be specified in the form \"name[=version]\"', required=True, nargs='+')
+  parser.add_argument('--operator', '-o', required=True, nargs='+', choices=VALID_OPERATORS,
```

Comment:
This breaks if the operator is specified with a version like `trino=0.3.2-pr157`:
```suggestion
  parser.add_argument('--operator', '-o', required=True, nargs='+',
```